uint32 msg_pub_cnt
uint32 msg_sub_cnt

int16 Steering_angle_feedback
uint16 APS_analog_feedback
int16 Tail_lamp
int16 Wiper
int16 Trun_lamp
int16 Mode
int16 Mode_feedback
int16 Error_status
int16 Desired_steering_angle
int16 status
int16 reserved
int16 TM_G_SEL_DISP
int16 accel
int16 mode_feedback
int16 ASM_state
int16 error_status
int16 Brake_actutator_position
int16 angle
int16 Brake_pedal

